extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -97,6 +98,10 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 			});
 		}
 
+		if (extra.sraEnabled) {
```

Comment:
Do we need to pass it through method calls?
Maybe we can have this flag in `adContext`?